app/services/concerns/author_extractor.rb

```diff
@@ -18,6 +18,6 @@ def author_from_xml(xml)
       acct   = "#{username}@#{domain}"
     end
 
-    FollowRemoteAccountService.new.call(acct)
+    FollowRemoteAccountService.new.call(acct, update_profile: update_profile)
```

Comment:
I'm not sure what you are talking about here… it's just the syntax for named parameters?